plugins/net_plugin/security_group_manager.cpp

```diff
@@ -0,0 +1,11 @@
+#include <eosio/net_plugin/security_group_manager.hpp>
+
+namespace eosio {
+   bool security_group_manager::update_cache(const uint32_t version, const participant_list_t& participant_list) {
+      if(version == version_)
+         return false;
+      version_ = version;
```

Comment:
Do we allow version going backward?

Reply:
The assert verbiage could use a bit of work.

Reply:
I think I fixed it